client/dashboard/agency-overview/index.tsx

```diff
@@ -1,22 +1,24 @@
+import { Header } from '@automattic/components/src/header';
```

Comment:
It's already exported from the index file, should this be just `from '@automattic/components`?

Reply:
We want to be more wary of what components we're using in the new dashboard and that's why we import them like this have the lint rule.

Reply:
Ooohh. TIL!